fiatconnect-api.md

```diff
@@ -1717,10 +1727,22 @@ A KYC schema containing personal data about a user, as well as documents such as
 
 The `selfieDocument` and `identificationDocument` fields should be base64 encoded binary blobs representing images.
 
-## 7.3. Initial Entity Support
+### 7.3.2. Fiat Account Schemas
 
-This section details the initial entity support for FiatConnect. In particular, the initial KYC and Fiat Account schemas that the FiatConnect specification
-should support for initial integrators. This section is pending further research and feedback from potential CICO providers.
+All Fiat Account Schemas supported by FiatConnect MUST contain the `name`, `institution`, and `fiatAccountType` fields. `name` is a friendly, user-definable name for the account.
+`institution` is the financial institution/organization the account is with, and `fiatAccountType` is a `FiatAccountTypeEnum` value, representing the type of fiat account this
+schema represents.
+
+####  7.3.2.1. `NGBankTransfer`
```

Comment:
if this is specific to Nigeria, would there be one type per country? Would that be scalable?

Reply:
If you want to go more generic, this could just be named `AccountNumber` since all the other fields are required.

Reply:
I'm somewhat concerned about the re-usability of these schemas... syntactically, this schema may be reusable for other geos. But semantically, this schema is for collecting this information for an account located in Nigeria. When a client receives this enum, it will know to prompt the user for a Nigerian account. That could mean a custom UI/integration that prompts for a Nigeria-specific account, or some restriction on what can be entered as `accountNumber` to comply with the expected syntax for Nigerian accounts.

Obviously, duplicating this schema for every country is not really scalable from a client's perspective (who would need to implement handlers for each possible schema). I'm not sure what the best approach is here. @cajubelt, any thoughts on this?

Reply:
one thought that comes to mind is adding a `country` field to the schema and using `allowedValues` to restrict that to `NG` for some use cases.

That way, the client could get clued in that only Nigerian bank accounts are allowed, and maybe surface that info somehow to the user (could even have a line of text saying: "Account must be from Nigerian bank", something like that)

Reply:
I'd also be fine with just renaming to `AccountNumber` to avoid over-engineering this.

Reply:
I like changing the schema name to `AccountNumber`. Having a `country` field that we restrict via `allowedValues` seems like a good option, albeit a bit hacky. I think it's the best solution we have that doesn't introduce a major redesign though, so I'm comfortable including it here.